ParabolInc/parabol
# feat(kudos): send kudos at the end of the retro

Fixes #9234

<img width="377" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/403b7b4b-79be-44ef-a8c0-598137fb0af7">

<img width="444" alt="image" src="https://github.com/ParabolInc/parabol/assets/466991/2954405b-0a89-4f15-afad-59672a756990">


**How to test:**

- Start a retro with few users
- Test various kudos
- End retro and see that users are received kudos notifications
- Test both anonymous and non anonymous reflections
